File: ios/AccountCenter/Models/Token.swift
import Foundation

struct Token: Codable {
    let accessToken: String
    let refreshToken: String
    let expiresIn: Int64
    let userId: Int64
    let accountId: String
    
    enum CodingKeys: String, CodingKey {
        case accessToken = "access_token"
        case refreshToken = "refresh_token"
        case expiresIn = "expires_in"
        case userId = "user_id"
        case accountId = "account_id"
    }
}

File: android/app/src/main/java/com/accountcenter/model/Token.kt
package com.accountcenter.model

import com.google.gson.annotations.SerializedName

data class Token(
    @SerializedName("access_token") val accessToken: String,
    @SerializedName("refresh_token") val refreshToken: String,
    @SerializedName("expires_in") val expiresIn: Long,
    @SerializedName("user_id") val userId: Long,
    @SerializedName("account_id") val accountId: String
)
